### PR TITLE
New topics for Banner and Expansion Panel

### PR DESCRIPTION
**Pull request by StefanIvanov**

### CODE REVIEW

File: en/components/banner.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Use the Banner Component to display a prominent message and optionally actions related to it. The Banner is visually identical to the [Ignite UI for Angular Banner Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/banner)
+
+## Banner Demo
+
+<img class="responsive-img" src="../images/banner_demo.png" srcset="../images/banner_demo@2x.png 2x" />
+
+## Message
+
+The Banner...
```
Comment (tosharky) [inline en/components/banner.md:17]:
```suggestion
The Banner's Message consists of an Icon and a Text field. It uses a smart layout on Sketch and a Stack on Adobe XD, allowing it to hide one or the other when set to ~No Symbol for Sketch users, and hiding the layer's visibility on Adobe XD.

Comment (tosharky) [reply en/components/banner.md:17]: Message
Message@2x

File: en/components/banner.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Actions
+
+The Banner...
```
Comment (tosharky) [inline en/components/banner.md:21]: The Actions consist of two Button components. If you need you can turn one or both off by setting them to ~No Symbol for Sketch or turning off the layer's visibility in Adobe XD.

Comment (tosharky) [reply en/components/banner.md:21]: Actions@2x
Actions

Comment (StefanIvanov) [reply en/components/banner.md:21]:
```suggestion
The Actions consist of two Button components. If you need you can turn one or both off by setting them to ~No Symbol for Sketch or turning off the layer's visibility in Adobe XD.
```

File: en/components/banner.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ The Banner has styling flexibility with the many options for the background, bor
 
 ## Usage
 
-When using a Banner...
+When using a Banner make sure your elements and texts are visible and readable on the background.
```
Comment (StefanIvanov) [inline en/components/banner.md:29]: look here and pick the 2-3  most applicable to our banner do-dont scenarios
https://material.io/components/banners#anatomy
about the images you may want to insert, just add the img tags with correct names and then archive and send them over to me to add them via source tree. there is such an image tag in the beginning of the topic

**Comment by StefanIvanov** [comment]
@tosharky take a look at the images, when you look at how they appear in the topic together it seems to me that we need to adjust some of the sizes to make everything more consistent and flowing

**Review by tosharky — APPROVED** [review]

File: en/components/expansion-panel.md [hunk 4]
```diff
@@ -1,106 +1,60 @@
...
-The Avatar comes with styling flexibility through the various options for the background color, as well as initials and icon colors, where applicable.
+The Expansion Panel comes with options for changing the both the Header's and Body's background colors, as well as text and icon colors.
 
-<img class="responsive-img" src="../images/avatar_styling.png" srcset="../images/avatar_styling@2x.png 2x" />
+<img class="responsive-img" src="../images/expansion_panel_styling.png" srcset="../images/expansion_panel_styling@2x.png 2x" />
 
 ## Usage
 
-When using an Avatar with initials or icons, pick their colors carefully to assure good contrast with the Avatar background color. Avoid colors with poor contrast like similar shades of the same color or combinations that create chromatic aberrations.
+When using an Expansion Panel make sure to use icons that give a proper idea of  its usage i.e. that there is more content available when the panel is expanded.
```
Comment (YoannaIvanova) [inline en/components/expansion-panel.md:47]:
```suggestion
When using an Expansion Panel, make sure to use icons that give a proper idea of its usage, i.e. that there is more content available when the panel is expanded.
```

**Review by YoannaIvanova — APPROVED** [review]

**Pull request merged**